roles/openshift_setup/tasks/patch_dnsmasq.yml

```diff
@@ -0,0 +1,81 @@
+- name: Ensure file is present
+  register: _stat_networking_info
+  ansible.builtin.stat:
+    path: /etc/ci/env/networking-info.yml
+
+- name: Load network parameters
+  when: _stat_networking_info.stat.exists
+  block:
+    - name: Slurp network-info.yml
+      register: _patch_dns_network_info_slurp
+      ansible.builtin.slurp:
+        src: "/etc/ci/env/networking-info.yml"
+
+    - name: Decode network-info.yml
+      ansible.builtin.set_fact:
+        _decoded_net_env: >-
+          {{
+            _patch_dns_network_info_slurp['content'] |
+            b64decode | from_yaml
+          }}
+
```

Comment:
I'd use the networking_mapper/load_env_definition.yml:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/networking_mapper/tasks/load_env_definition.yml

Reply:
Thanks, that's much nicer.

Reply:
Actually, we don't use the mapper expect in reproducer/VA jobs right?

Reply:
Any downside to using `networking-info.yml`? I believe that is available in all situations.

Reply:
the mapper is widely used in the reproducer. networking-info.yml is CI (deprecated) only - should really rely on the the mapper.

Reply:
@cjeanner shouldn't we use both then here? for nested and extracted crc I don't think we'll have the mapper setup when calling this, while for the reproducer we can use the mapper. IMO we could rely on networking-info for now and introduce the mapper as a follow-up